usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
This is an awfully generic and misleading name, compared to what it actually does.